Dockerfile

```diff
@@ -4,4 +4,5 @@ RUN git clone https://github.com/jt3k/chat-linker.git /usr/app/src && cd /usr/ap
 RUN cd /usr/app/src && npm install # TODO: Not stable enough; migrate to yarn
 COPY app-config.json /usr/app/src/app-config.json
 WORKDIR /usr/app/src
+CMD ["npm", "run build"]
```

Comment:
Nope. [According to the docs](https://docs.docker.com/engine/reference/builder/#cmd), "There can only be one CMD instruction in a `Dockerfile`".

Please replace that with `RUN npm run build` line right after `npm install` (see above).

Reply:
@ForNeVeR Fixed